index.html

```diff
@@ -0,0 +1,67 @@
+        <ul class="social-links">
+            <li>
+           <a href="#"> <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z" fill="#505F79"/>
+            </svg>
+           </a>
+            </li>
+            <li>
+            <a href="#">
+            <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M13.6448 3.03455C10.1561 3.03455 7.28906 5.86213 7.28906 9.37937V19.3104C7.28906 19.6897 7.59994 20.0001 7.97991 20.0001H11.5723C11.9523 20.0001 12.2632 19.6897 12.2632 19.3104V9.65524C12.2632 8.89661 12.8849 8.27592 13.6448 8.27592C14.4048 8.27592 15.0265 8.89661 15.0265 9.65524V19.3104C15.0265 19.6897 15.3374 20.0001 15.7174 20.0001H19.3098C19.6898 20.0001 20.0006 19.6897 20.0006 19.3104V9.37937C20.0006 5.89662 17.1682 3.03455 13.6448 3.03455Z" fill="#505F79"/>
+                <path d="M2.48705 0C1.10535 0 0 1.10345 0 2.48276C0 3.86207 1.10535 4.96552 2.48705 4.96552C3.86874 4.96552 4.97409 3.86207 4.97409 2.48276C4.97409 1.10345 3.86874 0 2.48705 0Z" fill="#505F79"/>
+                <path d="M4.28325 6.27588H0.690846C0.310881 6.27588 0 6.58622 0 6.96553V19.3104C0 19.6897 0.310881 20 0.690846 20H4.28325C4.66321 20 4.97409 19.6897 4.97409 19.3104V6.96553C4.97409 6.58622 4.66321 6.27588 4.28325 6.27588Z" fill="#505F79"/>
+            </svg>
+            </a>
+            </li>
+            <li>
+            <a href="#">    
+            <svg width="20" height="16" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z" fill="#505F79"/>
+            </svg>
+            </a>
+            </li>
```

Comment:
- [ ] Great job on wrapping your social media icons in the list 👏 👏 In addition to what you have done, I would suggest you add the styles to the list items so that they look similar to the Figma design. To be more specific, there are black points before each icon that should not be there so, it will be good if you remove them 😉 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/191998106-fbc255dc-6dc3-4279-9f7c-2a39cdc5d096.png)
**Figma design**
![image](https://user-images.githubusercontent.com/90222110/191998174-8703157c-d7d5-4b0c-8fa8-e50d4191dd03.png)

**HINT 💡** You can set the `list-style-type: none` for all the list item `<li>` wrapping an icon 😉 try it out and you will see 🏃